app/components/lg-lion-search.js

```diff
@@ -54,9 +61,31 @@ export default Ember.Component.extend({
       params['name'] = selectedName;
     }
 
+    if (selectedDob && !Ember.isBlank(selectedDob)) {
+      var searchRange = this.get('searchRange');
+
+      var thisYear = (new Date()).getFullYear();
+      var earlyYear = new Date(selectedDob);
+      var lateYear = new Date(selectedDob);
+
+      earlyYear.setFullYear(earlyYear.getFullYear() - searchRange);
+      lateYear.setFullYear(lateYear.getFullYear() + searchRange);
+
+      if (earlyYear.getFullYear() < (thisYear - 30)) {
+        earlyYear.setFullYear(thisYear - 75); // all old lions
```

Comment:
I get some cognitive whiplash with these variable names. I think it will be easier to read if variables with the name "year" are integers and with the name "date" are dates. If I'm reading this correctly, `thisYear` is an integer, but `earlyYear` and `lateYear` are dates. It is fairly confusing to see code like `earlyYear.getFullYear()`.